HW03/ekaterina_balabanovich/spec/balabanovich_hw03_task_2_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require_relative '../balabanovich_hw03_task_2'
+
+RSpec.describe 'Task2' do
+  describe '#log_format' do
+    context 'when logs are not empty' do
+      let(:logs) do
+        <<~LOGS
+          10.6.246.103 - - [23/Apr/2018:20:30:39 +0300] 'POST /test/2/messages HTTP/1.1' 200 48 0.0498
+          10.6.246.101 - - [23/Apr/2018:20:30:42 +0300] 'POST /test/2/run HTTP/1.1' 200 - 0.2277
+          2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>
+          10.6.246.101 - - [23/Apr/2018:20:31:39 +0300] 'POST /test/2/messages HTTP/1.1' 200 48 0.0290
+        LOGS
+      end
+
+      let(:error_logs) do
+        <<~LOGS
+          2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>
+          2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>
+          2018-04-23 20:30:42: SSL ERROR, peer: 10.6.246.101, peer cert: , #<Puma::MiniSSL::SSL: System error: Undefined error: 0 - 0>
+        LOGS
+      end
+
+      it 'returns a string in a specific format' do
+        expect(log_format(logs).first).to eq('23/Apr/2018:20:30:39 +0300 FROM: 10.6.246.103 TO: /TEST/2/MESSAGES')
+      end
+
+      it 'skips errors cases' do
```

Comment:
this name isn't full enough 
```suggestion
      it 'skips errors cases and returns an empty array' do
```